frontend/components/Meetups/useSortMeetups.js

```diff
@@ -0,0 +1,55 @@
+export function useSortMeetups (selectedMeetups, sortType) {
+  const oldMeetups = []
+  const futureMeetups = []
+  const today = new Date()
+  let sortMeetups = []
```

Comment:
Maybe `sortedMeetups` would be more verbose? In the name of the function too?

Reply:
you're right, I will change it :)